auto.def

```diff
@@ -915,7 +897,6 @@ make-config-header config.h -auto $auto_rep -bare $bare_rep -str $str_rep
 # Generate doc/neomutt.1
 define bindir [get-define BINDIR]
 define docdir [get-define PKGDOCDIR]
```

Comment:
Aren’t `bindir` and `docdir` obsolete now, since `neomutt.1` is no longer generated here and
`Makefile.autosetup` defines them?